lib/awspec/type/acm.rb

```diff
@@ -0,0 +1,15 @@
+module Awspec::Type
+  class Acm < Base
+    def resource_via_client
+      @resource_via_client ||= find_certificate(@display_name)
+    end
+
+    def id
+      @id = resource_via_client.certificate_arn if resource_via_client
+    end
+
+    def issued?
```

Comment:
Could you use `define_method` like https://github.com/k1LoW/awspec/blob/master/lib/awspec/type/ebs.rb#L23 ?

( see http://docs.aws.amazon.com/sdkforruby/api/Aws/ACM/Client.html#describe_certificate-instance_method and check `resp.certificate.status` )